src/coreclr/jit/importer.cpp

```diff
@@ -12557,10 +12554,13 @@ GenTree* Compiler::impCastClassOrIsInstToTree(
     GenTree* condTrue;
     if (isCastClass)
     {
```

Comment:
Is there a reason why we are not doing the PGO profiling for the CastAny helpers?

It would be nice to note that reason in impIsCastHelperMayHaveProfileData/impIsCastHelperMayHaveProfileData.

Reply:
I can look on enabling those separately if you don't mind - probably we can benefit from them a lot, will experiment.

Reply:
Yes, that's fine. It is just something that I have run into while reviewing this change.